src/infrastructure/acknowlegement.rs

```diff
@@ -62,7 +62,8 @@ mod test {
         handler.seq_num = 40;
         handler.process_outgoing(vec![1, 2, 4].as_slice());
 
-        handler.process_incoming(40);
+        static ARBITRARY: u16 = 23;
```

Comment:
does this add really that much of a difference?

Reply:
I had to add two arguments, and it made sense to me this way